### PR TITLE
Local docs

### PR DESCRIPTION
**Pull request by Rich-Harris**
Step 4 of the [checklist](https://github.com/sveltejs/kit/issues/3696)

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Review by benmccann — APPROVED** [review]

File: sites/kit.svelte.dev/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"version": "0.0.1",
 	"scripts": {
 		"dev": "svelte-kit dev",
-		"build": "svelte-kit build",
+		"build": "svelte-kit build --verbose",
```
Comment (Conduitry) [inline sites/kit.svelte.dev/package.json:6]: Is this change intended? TBH I didn't even know there was a `--verbose` option to `svelte-kit build`.

Comment (Rich-Harris) [reply sites/kit.svelte.dev/package.json:6]: i prefer having it, because then i can see which pages are getting prerendered
